src/main/java/Deadlines.java

```diff
@@ -0,0 +1,12 @@
+public class Deadlines extends Task{
```

Comment:
I suggest adding comments into your code so that it would be easier to understand